web/console-extensions.json

```diff
@@ -109,6 +109,33 @@
       }
     }
   },
+  {
+    "type": "console.tab",
+    "properties": {
+      "contextId": "dev-console-observe",
+      "name": "%plugin__monitoring-plugin~Silences%",
+      "href": "silences",
+      "component": {
+        "$codeRef": "SilencesPage"
```

Comment:
Is there a reason this code ref needs to be different from the MonitoringUI entrypoint that the rest use?

Reply:
I updated this to use the MontioringUI entry point that the rest use in the latest push.